pom.xml

```diff
@@ -828,14 +838,21 @@
                         <hazelcast.phone.home.enabled>false</hazelcast.phone.home.enabled>
                     </systemPropertyVariables>
                 </configuration>
+                <dependencies>
+                    <dependency>
+                    <groupId>org.ow2.asm</groupId>
+                        <artifactId>asm</artifactId>
+                        <version>${asm.version}</version>
+                    </dependency>
+                </dependencies>
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-failsafe-plugin</artifactId>
                 <version>2.22.2</version>
                 <configuration>
                     <redirectTestOutputToFile>${test.redirectOutput}</redirectTestOutputToFile>
-                    <argLine>-Xmx1024m -DjvmId=${surefire.forkNumber} ${failsafe.jacoco.args}</argLine>
+                    <argLine>-Xmx1024m -DjvmId=${surefire.forkNumber} --add-modules java.sql,java.xml ${failsafe.jacoco.args}</argLine>
```

Comment:
These arguments are commented out in the build step above (--add-modules java.sql,java.xml ), are they needed here?

Reply:
They will be when we start compiling for Java11.